roles/openshift_master_upgrade/vars/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
+openshift_master_config_file: "{{ openshift_master_config_dir }}/master-config.yaml"
+openshift_master_scheduler_conf: "{{ openshift_master_config_dir }}/scheduler.json"
```

Comment:
It seems like we should have an common role to set these variables, since both roles already rely on `openshift_master_facts`, maybe that is a good place for it.